homeassistant/components/totalconnect/alarm_control_panel.py

```diff
@@ -114,16 +116,40 @@ def update(self):
 
     def alarm_disarm(self, code=None):
         """Send disarm command."""
-        self._client.disarm(self._location_id)
+        if self._client.disarm(self._location_id) is not True:
+            persistent_notification.async_create(
```

Comment:
Do we need a notification here?

Can you instead

`raise HomeAssistantError(f"TotalConnect failed to disarm {self._name}.")`